lib/user_interface.rb

```diff
@@ -84,16 +76,18 @@ def play
   end
 
   def new_score
-    puts message(:new_score).red.bold \
+    puts ' New Score !!'.red.bold \
       if @board.add(@name, @player.tries, @player.time)
+    PressAnyKey.new
+    UserInterface.new
   end
 
   def check_winner
```

Comment:
Ah, I see.  Yes, the magician is indicating the player!  "Wut!?  Are you a magician?!"

But then it unfortunately also knows about business logic, as it then goes to writing the score.